tests/extras/extensions/test_ipython.py

```diff
@@ -53,9 +58,7 @@ def my_register_pipeline():
             "kedro.framework.project._ProjectPipelines._get_pipelines_registry_callable",
             return_value=my_register_pipeline,
         )
-        mocker.patch("kedro.framework.project.settings.configure")
-        mocker.patch("kedro.framework.session.session.validate_settings")
-        mocker.patch("kedro.framework.session.KedroSession._setup_logging")
```

Comment:
@AntonyMilneQB Are these actually needed? I found that they are not in the `test_load_kedro_objects_extra_args` test and removing them seems to have no effect, I am not sure why it is needed.

Reply:
Probably not needed 👍